src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -692,6 +692,29 @@ function setupMobileSidebarKeyboardHandlers() {
   });
 }
 
+/**
+ * When the page loads or the window resizes check all elements with
+ * [data-tabindex="0"], and if they have scrollable overflow, set tabIndex = 0.
+ */
+function setupLiteralBlockTabStops() {
+  const updateTabStops = () => {
+    document.querySelectorAll('[data-tabindex="0"]').forEach((el) => {
+      el.tabIndex = el.scrollWidth > el.clientWidth ? 0 : -1;
```

Comment:
Right now, I'm only checking for scrollable content in the x, or horizontal, direction because that's all I've ever seen. But do we know if we ever have code blocks that scroll in the y direction?

Reply:
I don't think we do, or I don't see any reason to have some. The only thing I ce see that would be vertically scrollable would be inclusions like widgets/iframe from things like jupyter-lite-sphinx.

Reply:
I decided to extend this line to check for y-overflow too. My reasoning is that the theme should be work robustly and it would be fairly easy for some documentation site to create a class like `.height-10em` that they apply to code blocks.